homeassistant/components/rflink.py

```diff
@@ -253,26 +265,31 @@ class RflinkDevice(Entity):
     """
 
     platform = None
-    _state = STATE_UNKNOWN
+    _state = None
     _available = True
 
-    def __init__(self, device_id, hass, name=None, aliases=None, group=True,
-                 group_aliases=None, nogroup_aliases=None, fire_event=False,
+    def __init__(self, initial_event, device_id, name=None, aliases=None,
```

Comment:
Since `initial_event` is optional we could make it a keyword argument and default it to `None`. Probably put it after `device_id` and before `name`.